### ISSUE TITLE
Can't seem to pass custom argument

### ISSUE THREAD

**Issue by jmbockhorst** [danielgjackson/vscode-maximize-fix#1]
Thanks for making this! It works great. 

Just one thing, I was trying to get it to work for VSCode Insiders, but the arg doesn't seem to get passed.

```
 .\vscode-maximize-fix.exe "Visual Studio Code - Insiders"
ERROR: No windows found matching suffix: Visual Studio Code
```

**Comment by danielgjackson** [comment]
Ah sorry, the executable build must've been just before I added that feature.  The current release should fix it (and even has a shiny icon!)

I've also just made this into a VS Code Extension but, thinking about it, it will also need the title customizing for the Insiders build.

**Comment by jmbockhorst** [comment]
Could you just make the extension check for both stable and insiders?

**Comment by danielgjackson** [comment]
Yes, that's more sensible. 

Fixed in commit fb86e0a85092e1a189d74412d9812a23740e425d -- release 0.0.3 of the extension.  

The executable (binary release 1.2) defaults to "Visual Studio Code" and "Visual Studio Code - Insiders", and takes any number of command-line arguments to match against custom window titles.
